index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <a href="#" id="logo">LOGO</a>
+    <nav>
+      <ul id="nav-menu">
+        <li class="nav-link"><a href="#works">Portfolio</a></li>
+        <li class="nav-link"><a href="#about-myself">Myself</a></li>
+        <li class="nav-link"><a href="#form-section">Contact</a></li>
+      </ul>
+      <div id="mobile-navbar">
+        <div id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+       </div>
+    </nav>
+  </header>
+
+  <section id="headline">
+    <div id="headline-content">
+      <h1 class="header">Tanusri Here from India <br> Have a good day!</h1>
+      <p class="portfolio-description">I am a web developer. Need Web Development ----- Get in touch with me
+      </p>
+      <h2 class="lets-connect">CONNECT ME</h2>
+      <ul class="social-media-links">
+        <li> <a href="https://www.linkedin.com/in/tanusri-ghosh-2a56b814"> LinkedIn Account </a> </li>
+	      <li> <a href="https://twitter.com/chuaghosh25"> Twitter Account </a> </li>
+	      <li> <a href="https://www.google.com/tghosh25@gmail.com"> Google Account </a> </li>
+      </ul>
+    </div>
+  </section>
```

Comment:
Well done setting up the Html skeleton correctly, you have used semantic tags properly 👏🏾🎆 

- Kindly fix the layout of your Headline section, it's barely visible. Be sure that the background is also visible as per [FIGMA design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

Your Headline:

![image](https://user-images.githubusercontent.com/38088818/185992557-f460a7b2-43d8-4235-8a08-ca3d1b63eeed.png)

Expected layout:

![image](https://user-images.githubusercontent.com/38088818/185993019-e5475c0e-924e-4281-a64c-8d4975aa101c.png)